### PR TITLE
Fix invalid index mapping in README.md

### LINKED ISSUE #153: Index mapping shown in README.md is invalid
I think the given example in "README.md" is not valid, e.g. the snippet

```
"file" : {
    "properties" : {
      "content_type" : {
          "type" : "string",
          "analyzer" : "not_analyzed",
          "store" : "yes"
      },
```

references an analyser with the name "not_analyzed". The intention was probably the following

```
"index" : "not_analyzed"
```

Currently the following error message is shown

```
{
   "error": {
      "root_cause": [
         {
            "type": "mapper_parsing_exception",
            "reason": "analyzer [not_analyzed] not found for field [content_type]"
         }
      ],
      "type": "mapper_parsing_exception",
      "reason": "analyzer [not_analyzed] not found for field [content_type]"
   },
   "status": 400
}
```

### PR DESCRIPTION
**Pull request by sgoeschl**
Using the index mapping straight from ES

Closes #153.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -576,98 +602,114 @@ PUT docs
 # Create the mapping
 PUT docs/doc/_mapping
 {
-  "doc" : {
-    "properties" : {
-      "content" : {
-        "type" : "string",
-        "store" : "yes",
-        "analyzer" : "french"
```
Comment (dadoonet) [inline README.md]: Can you please add back the french analyzer here?

File: README.md [hunk 2]
```diff
@@ -576,98 +602,114 @@ PUT docs
...
       },
-      "file" : {
-        "properties" : {
-          "content_type" : {
-              "type" : "string",
-              "analyzer" : "not_analyzed",
-              "store" : "yes"
+      "content": {
+        "type": "string",
+        "store": true
```
Comment (dadoonet) [inline README.md]: French analyzer has been removed in this new version

**Comment by dadoonet** [comment]
I left a comment.

Would be nice if you could fix then rebase on master then squash your changes.

Thanks!

**Comment by dadoonet** [comment]
I forgot. While you are squashing, could you use a commit message like:

```
Fix invalid Index mapping in README.md

Closes #153.
```

Thanks!

**Comment by sgoeschl** [comment]
I hate to say this but I'm stuck

I squashed my commits as shown below

```
fscrawler> git ls
ae473fa (HEAD -> feature/FSCRAWLER-153) Fix invalid Index mapping in README.md [Siegfried Goeschl]
c6f3b72 (origin/master, origin/HEAD, master) Merge branch 'master' of https://github.com/sgoeschl/fscrawler [Siegfried Goeschl]
26d2d73 (upstream/master) FSCrawler is using local dirs instead of remote dirs [David Pilato]
```

but git push says

```
fscrawler> git push
To https://github.com/sgoeschl/fscrawler.git
 ! [rejected]        feature/FSCRAWLER-153 -> feature/FSCRAWLER-153 (non-fast-forward)
error: failed to push some refs to 'https://github.com/sgoeschl/fscrawler.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

**Comment by dadoonet** [comment]
You have to force push when you rewrite an history

**Comment by sgoeschl** [comment]
Done

**Comment by dadoonet** [comment]
I think that if you rebase on master branch, the first 2 commits should disappear and we will have only one commit within this PR.

If you don't succeed, I can do it on my end though.

**Comment by sgoeschl** [comment]
Don't get it to work - please take over :-)

**Comment by dadoonet** [comment]
Thank you! I merged it with 52e65d1b379a6b8fa32325b69c8d837a71c11ac8.

**Pull request closed without merging**